olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -1121,23 +1121,23 @@ spec:
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-receiver-rhel8@sha256:05217617dd8721fe760f1d0200dbf567d277c627d68f58bb2050c642ccaef785"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-receiver-rhel8@sha256:c2c6cf3d8bec5afa07c461a5cf61e0c4d43fe0eb402f371687e4756142634ee6"
                       - name: "KAFKA_IMAGE_kafka-channel-dispatcher__kafka-channel-dispatcher"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-dispatcher-rhel8@sha256:30c8cbb6228b8ff4233fc3bd272d383d27678afb690c501555f020327394d874"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-dispatcher-rhel8@sha256:84bc7f1a1ee8347035587085544fd392f3feaaf37b4a1f251ad74b965350a310"
                       - name: "KAFKA_IMAGE_kafka-controller__controller"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-kafka-controller-rhel8@sha256:10422d207b1bbf3b715c131d25cacdec26138feb32e82c6088d44b0157f070a4"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-kafka-controller-rhel8@sha256:7f4f35c0ff6a19405a289128e3778642f94db27164b6492389c71b1e355d12aa"
                       - name: "KAFKA_IMAGE_kafka-sink-receiver__kafka-sink-receiver"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-receiver-rhel8@sha256:05217617dd8721fe760f1d0200dbf567d277c627d68f58bb2050c642ccaef785"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-receiver-rhel8@sha256:c2c6cf3d8bec5afa07c461a5cf61e0c4d43fe0eb402f371687e4756142634ee6"
                       - name: "KAFKA_IMAGE_kafka-source-dispatcher__kafka-source-dispatcher"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-dispatcher-rhel8@sha256:30c8cbb6228b8ff4233fc3bd272d383d27678afb690c501555f020327394d874"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-dispatcher-rhel8@sha256:84bc7f1a1ee8347035587085544fd392f3feaaf37b4a1f251ad74b965350a310"
                       - name: "KAFKA_IMAGE_kafka-webhook-eventing__kafka-webhook-eventing"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-webhook-kafka-rhel8@sha256:cb71952b05b016efcd2306f1134247a232926be09356fa37a64ac1a6c8c998fa"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-webhook-kafka-rhel8@sha256:2db14cd4960bf025318d7a8c0d25bad132c22be4807e31c3a85c73d857d7d8f0"
                       - name: "KAFKA_IMAGE_kafka-controller-post-install__post-install"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-post-install-rhel8@sha256:c19f19d56daf95f8c8a773b0535230cb223c74f29ad2f8d968d1f994dccb0cba"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-post-install-rhel8@sha256:c812871cac20d7b23bcec5ff88120e5a2e5099f46fdd17a87f7cfd481e423bf6"
```

Comment:
Those are the images after the quarkus update (https://github.com/openshift-knative/eventing-kafka-broker/commit/b9dcd39000997d8478accd87c76e3d3ade6759f9)
/lgtm
/approve